app/assets/javascripts/discourse/app/mixins/card-contents-base.js

```diff
@@ -188,104 +190,75 @@ export default Mixin.create({
-                  (target.offset().top - $("html").scrollTop());
-                if (verticalAdjustments > 0) {
-                  position.bottom += 48;
-                }
-                position.top = "unset";
-              }
-            }
-
-            const avatarOverflowSize = 44;
-            if (isDocked && position.top < avatarOverflowSize) {
-              position.top = avatarOverflowSize;
-            }
-
-            $(this.element).css(position);
-          }
-        }
-
-        if (this.site.mobileView) {
-          $(".card-cloak").removeClass("hidden");
-          let position = target.offset();
-          position.top = "10%"; // match modal behaviour
-          position.left = 0;
-          $(this.element).css(position);
-        }
-        $(this.element).toggleClass("docked-card", isDocked);
-
-        // After the card is shown, focus on the first link
-        //
-        // note: we DO NOT use afterRender here cause _positionCard may
-        // run afterwards, if we allowed this to happen the usercard
-        // may be offscreen and we may scroll all the way to it on focus
-        if (event.pointerId === -1) {
-          discourseLater(() => {
-            const firstLink = this.element.querySelector("a");
-            firstLink && firstLink.focus();
-          }, 350);
-        }
+      if (!target) {
+        return;
+      }
+
+      if (this.site.desktopView) {
+        const avatarOverflowSize = 44;
+        this._popperReference = createPopper(target[0], this.element, {
+          placement: "right",
+          modifiers: [
+            {
+              name: "preventOverflow",
+              options: {
+                padding: {
+                  top: headerOffset() + avatarOverflowSize,
+                  right: 10,
+                  bottom: 10,
+                  left: 10,
+                },
+              },
+            },
+            { name: "eventListeners", enabled: false },
+            { name: "offset", options: { offset: [10, 10] } },
+          ],
+        });
+      } else {
+        $(".card-cloak").removeClass("hidden");
+        this._popperReference = createPopper(target[0], this.element, {
+          modifiers: [
+            { name: "eventListeners", enabled: false },
+            {
+              name: "computeStyles",
+              enabled: true,
+              fn({ state }) {
+                // mimics our modal top of the screen positioning
+                state.styles.popper = {
+                  ...state.styles.popper,
+                  position: "fixed",
+                  left: `${
+                    (window.innerWidth - state.rects.popper.width) / 2
+                  }px`,
+                  top: "10%",
+                  transform: "translateY(-10%)",
+                };
+
+                return state;
+              },
+            },
+          ],
+        });
+      }
+
+      $(this.element).toggleClass("docked-card", this.isDocked);
```

Comment:
Just a bit of dejq while we're at it 😃 

```suggestion
      this.element.classList.toggle("docked-card", this.isDocked);
```